.github/actions/docker/action.yaml

```diff
@@ -25,21 +25,24 @@ inputs:
   cargo_profile:
     description: Cargo build profile, i.e release or dev
     default: dev
-  bucket:
-    description: S3 bucket to use for caching, must match runner define in `runs-on`
-    default: multi-runner-cache-x1xibo9c
-  region:
+  cache_bucket:
+    description: S3 bucket to use for caching (both sccache and layer cache)
+    required: true
+  cache_region:
     description: S3 bucket region
     required: true
-  aws_access_key_id:
-    description: AWS access key ID
+  cache_endpoint:
```

Comment:
It's a bit confusing that we use vars from inputs for sccache but for layer cache we are reading the config file

Reply:
Refactored a bit